modules/TestUtils.js

```diff
@@ -20,6 +20,12 @@ export const isObject = (object) =>
   object && !isArray(object) && typeof object === 'object'
 
 /**
+ * Returns true if the given object is an empty object.
+ */
+export const isEmptyObject = (object) =>
+  isObject(object) && Reflect.ownKeys(object).length === 0
```

Comment:
Those tests should be conditionally skipped, so that they run in environments with native Symbols, and not in envs without them.

Also `Object.getOwnPropertyNames` will not be available in an ES3 context, so you may want to use the `object-keys` module in that case.